src/Components/Presentational/book.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
```

Comment:
### [Optional]
we could adjust the file  structure, it should be `src/Component/book.js`

![image](https://user-images.githubusercontent.com/31897434/127382776-fa0b9f91-0cb6-4dbe-8904-7da74a12e4b9.png)